health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SetupCampaign.js

```
@@ -769,34 +1021,34 @@ const SetupCampaign = ({ hierarchyType, hierarchyData }) => {
+              type={"actionButton"}
+              options={actionbarOptions}
+              label={t("ES_COMMON_TAKE_ACTION")}
+              variation={"primary"}
+              style={{ width: "14rem" }}
+              optionsKey={"name"}
+              isSearchable={false}
+              t={t}
+              onOptionSelect={(option) => {
+                onActionSelect(option?.code);
+              }}
+            ></Button>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)